lib/matplotlib/lines.py

```diff
@@ -790,6 +790,9 @@ def draw(self, renderer):
                     cap = self._dashcapstyle
                     join = self._dashjoinstyle
                 else:
+                    print(tpath.vertices.size)
+                    tpath = affine.inverted().transform_path(affine.transform_path(tpath).cleaned(simplify=True))
+                    print(tpath.vertices.size)
```

Comment:
Obviously print statements will be removed and tests will be added.

Testing in general will be a huge pain, since nearly every single image will fail the current image comparisons. We may have to change the default value of `path.simplify` to False to avoid this...